pyproject.toml

```diff
@@ -1,3 +1,6 @@
 [build-system]
 requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
```

Comment:
I'm wondering if it would be a better practice to avoid this project level event loop config to use `@pytest.mark.asyncio(loop_scope="session")` for each test case?

Reply:
Yes, I think being explicit with scopes with fixtures is a good practice and we should keep doing it for our fixtures. 

I think it should be fine to have this config. It is not so we can avoid setting explicit scopes, but just to have the default scope when the scope is not specified as `pytest-asyncio` will eventually change it "function" by default - in the end we get the same. This way we only explicitly say it so we don't get the warning telling us our default scope will be changed.

Reply:
I agree with @yanxi0830 -- I don't think we should set a default here. we should silence the warnings explicitly because the scope of a fixture is pretty important to think through. A function scope is good for isolation but cannot be too expensive and sometimes just not intended.